src/sentry/integrations/slack/requests/action.py

```diff
@@ -46,6 +46,8 @@ def callback_data(self) -> JSONData:
         if self.data["type"] == "block_actions":
             if self.data.get("view"):
                 return json.loads(self.data["view"]["private_metadata"])
+            elif self.data["container"].get("is_app_unfurl"):
+                return json.loads(self.data["app_unfurl"]["blocks"][0]["block_id"])
```

Comment:
this is used to carry out actions on unfurled issues in slack; i will try to add tests for this but i don't believe there are any existing ones for when block kit isn't enabled